packages/odyssey-react/src/components/Infobox/Infobox.theme.ts

```diff
@@ -0,0 +1,33 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import type { ThemeReducer } from "@okta/odyssey-react-theme";
+
+export const theme: ThemeReducer = (theme) => ({
+  // Root
+  FontLineLengthMax: theme.FontLineLengthMax,
+  SpacePadding: theme.SpaceRemM,
```

Comment:
@jeffbelser-okta Naming note: we expect the padding to be uniform, so I haven't marked anything more specific than `Padding`, below I've specified the lone Margin to be themed by adding `BlockEnd`.

Reply:
hmmm I don't like that this exposes our internal CSS implementation (i.e. using the shorthand and a more specific declaration cascade override with `&:not(:last-child)`) but functionally its all correct. I don't feel too strongly this works fine!